repository/src/main/java/org/alfresco/repo/content/transform/TransformerDebug.java

```diff
@@ -210,57 +210,6 @@ public String getRenditionName()
         }
     }
 

```

Comment:
The UnavailableTransformer was only needed for Legacy transformers. There is lots of other code in this class that could be removed, but I do not wish to destabilize what is working.